### ISSUE TITLE
[Programmatic Access – Make Code - Language] Ensures every ARIA input field has an accessible name (#availablelocales)

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3662]
**User Experience**:
If any aria input fields does not have accessible name than screen reader user will not be able to find out what kind of information is needed in the form fields and will face difficulties while navigating on the page.

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Language " option and select it.
7. Navigate through all the controls appearing under "Select Language" dialog.
8. Run accessibility insights for web tool and select fast pass.
9. Verify the issue.

**Actual Result:**
Aria input fields does not have accessible name under 'Select Language' pop-up window.

**Expected Result:**
Aria input fields should have accessible name under 'Select Language' pop-up window.

**Issue:** Ensures every ARIA input field has an accessible name (aria-input-field-name)

**Target application:** Microsoft MakeCode for micro:bit
**Element path:** #availablelocales

**Snippet:** <div id="availablelocales" class="ui cards centered" role="listbox">

**How to fix:** 
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute or the title attribute is empty

This accessibility issue was found using Accessibility Insights for Web 2.22.0 (axe-core 4.0.2), a tool that helps find and fix accessibility issues.

**Note:**
Same issue appear under Chrome & Firefox.

**WCAG Reference:**
[WCAG 4.1.2 - Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG4.1.2; #Win10-Edge(Chromium)

![WCAG4 1 2_Language](https://user-images.githubusercontent.com/72871089/98902044-ee097600-24da-11eb-90b9-3764744d1559.jpg)
